java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/MacAddressTest.java

```diff
@@ -22,12 +23,18 @@
             {(byte) 255, (byte) 255, (byte) 255, (byte) 255, (byte) 255, (byte) 255 }
     };
 
-    String[] testStrings = {
+    String[] testColonStrings = {
             "01:02:03:04:05:06",
             "80:00:00:00:00:01",
-            "ff:ff:ff:ff:ff:ff"
+            "ff:ff:ff:ff:ff:ff",
```

Comment:
Didn't know you could do that in Java these days (the "," at the end of an array). Nice to know!